_layouts/project.html

```diff
@@ -10,9 +10,26 @@
 
         <div class="content">
             <h1>{{page.name}}</h1>
-
-            This should be a Project page.
-            {{content}}
+            {% if page.members %}
+            <h6>Lab members on project: {{page.members}}</h6>
+            {% endif %}
+            <table style="width:100%">
+                <tr>
+                    {% if page.video %}
+                    <td style="text-align: center;"> 
```

Comment:
pls don't use inline styles
Make a _sass/_projects.scss file and include it in style.scss